ambry-coordinator/src/test/java/com.github.ambry.coordinator/CoordinatorTest.java

```diff
@@ -1034,13 +1034,13 @@ public void simpleOneDCFourNodeOneDiskTwoPartition()
     simple(getClusterMapOneDCFourNodeOneDiskTwoPartition());
   }
 
-  @Test
```

Comment:
We discussed that and decided to keep them in our internal backlog ticket, since we have not started creating github issues.